config/core/resources/old_pullsubscription.yaml

```diff
@@ -103,15 +103,11 @@ spec:
                     kind:
                       type: string
                       minLength: 1
+                    namespace:
+                      type: string
```

Comment:
Not sure about the purpose of leaving this old yaml. Could you explain?

Reply:
after the 0.15 cut I think we can remove it. It was due to a migration of the api group @Harwayne did in order to follow the proper deprecation policy.